docs/sql-reference/sql-statements/data-definition/CREATE TABLE.md

```diff
@@ -270,30 +270,51 @@ For more information, see [Data distribution](../../../table_design/Data_distrib
+  - **When you create a table, you must specify the bucketing columns**.
+  - The values of bucketing columns cannot be updated.
+  - Bucketing columns cannot be modified after they are specified.
+  - Since StarRocks 2.5.7, you do not need to set the number of buckets when you create a table. StarRocks automatically sets the number of buckets. If you want to set this parameter, see [Determine the number of buckets](../../../table_design/Data_distribution.md#determine-the-number-of-buckets).
 
-- High cardinality column such as ID
-- Column that is often used as a filter in queries
+- Random bucketing (since v3.1)
 
-If such a column does not exist, you can determine the bucketing column according to the complexity of queries.
+  For data in a partition, StarRocks distributes the data randomly across all buckets, which is not based on specific column values. And if you want the StarRocks to automatically determine the number of buckets, you do not need to specify any bucketing information. If you choose to manually specify the number of buckets, the syntax is as follows:
 
-- If the query is complex, we recommend that you select a high cardinality column as the bucketing column to ensure balanced data distribution among buckets and improve cluster resource utilization.
-- If the query is relatively simple, we recommend that you select the column that is often used as the query condition as the bucketing column to improve query efficiency.
+  ```SQL
+  DISTRIBUTED BY RANDOM BUCKETS <num>
+  ```
 
-If partition data cannot be evenly distributed into each tablet by using one bucketing column, you can choose multiple bucketing columns (at most three). For more information, see [Choose bucketing columns](../../../table_design/Data_distribution.md).
+  However, note that random bucketing may not be suitable for scenarios that involve querying and aggregating based on specific columns. In such cases, hash bucketing may be more appropriate as it can store similar data in the same bucket, facilitating data access and processing.
 
-**Precautions**:
+  **Precautions**
+  - You can not use random bucketing to create a Primary Key table, a Unique Key table, or an aggregated table.
+  - You can not specify a table bucketed randomly to belong to a Colocation Group.
+  - Spark Load can not be used to load data into tables bucketed randomly.
 
-- **When you create a table, you must specify the bucketing columns**.
-- The values of bucketing columns cannot be updated.
-- Bucketing columns cannot be modified after they are specified.
-- Since StarRocks 2.5, you do not need to set the number of buckets when you create a table. StarRocks automatically sets the number of buckets. If you want to set this parameter, see [Determine the number of tablets](../../../table_design/Data_distribution.md#determine-the-number-of-tablets).
+  For more information, [Random bucketing](../../../table_design/Data_distribution.md#random-bucketing-since-v31).
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/64a0dcf5-6f9d-4f3e-a0d4-0ac395814980)
the verb "see" is missing here